in-depth/submission_guidelines.md

```diff
@@ -3,31 +3,31 @@ layout: docs
 title: Theme & Plugin Submission Guidelines
 headline: Theme & Plugin Submission Guidelines
 description: |
-  Submitting your Theme or Plugin for inclusion on the website is very easy.<br>
+  Submitting your theme or plugin for inclusion on the website is very easy.<br />
```

Comment:
Don't worry, I won't search for it, just one of the things I change when I stumble over it when editing a file anyway :laughing: